src/ralph/backends/storage/fs.py

```diff
@@ -62,7 +62,7 @@ def list(self, details=False, new=False):
         logger.debug("Found %d archives", len(archives))
 
         if new:
-            archives = set(archives) - set(self.get_command_history(self.name, "fetch"))
+            archives = set(archives) - set(self.get_command_history(self.name, "read"))
```

Comment:
That's tricky. Should we support legacy history?

Reply:
Or provide an history migration script.

Reply:
As decided, we shall provide a migration script in the documentation. 

For memory, the alternative was to provide an additional `migrate` command to cli. Also, it was decided NOT to support legacy.